helm-frontend/project_metadata.json

```diff
@@ -66,6 +66,12 @@
 		"releases": ["v1.1.0", "v1.0.0"]
 	},
 	{
+		"title": "SEA-HELM",
+		"description": "Assessment of large language models across various tasks, emphasizing Southeast Asian languages",
+		"id": "seahelm",
+    "releases": ["v1.0.0"]
```

Comment:
Do you prefer the URL to be https://crfm.stanford.edu/helm/sea-helm/ or https://crfm.stanford.edu/helm/seahelm/ ? Currently it is the former, but I can change it to be the latter.